### ISSUE TITLE
Improve CLI utils argument+parameters validation

### ISSUE THREAD

**Issue by dewyatt** [riboseinc/rnp#404]
## Description

Related: https://github.com/riboseinc/rnp/issues/323

We should try to improve our validation of arguments and parameters within our CLI utilities when time permits.
There are at least a few things that will *silently* accept invalid parameter values, for example.

If we, the developers, are having trouble with this, then end-users likely will too.

## Steps to Reproduce

1. Example: https://github.com/riboseinc/rnp/pull/368#issuecomment-323357369 (`--format` specifies the key listing format, `--keystore-format` is the key storage format)
2. I know there are other similar ones, I recall hitting a few in the past and getting thoroughly confused.

## Expected Behavior

For example, `--format` should only accept certain parameter values (`human`, `mr`). If an invalid value is used, an error should be printed and the utility should exit with an unsuccessful exit code. (This argument is poorly named anyways, but that's a separate discussion)

## Actual Behavior

Invalid things are silently ignored, which can be a bit maddening.

**Comment by ni4** [comment]
@dewyatt Agree, we have quite basic command-line parameters validation in CLI.
I can take this one when I have time.
